doc/example_odMLs/ex_1.rdf

```diff
@@ -1,142 +1,114 @@
+    odml:name "NoCrewMembers" ;
+    odml:uncertainty "1" .
+
+<https://g-node.org/projects/odml-rdf#4b2d0e9d-6352-4676-97d7-fc0a795e83a9> a odml:Section ;
+    odml:definition "Information on the crew" ;
+    odml:name "s21" ;
+    odml:type "crew" .
+
+<https://g-node.org/projects/odml-rdf#635035cc-1c05-4e44-a4b7-0fb3d6295abf> a odml:Property ;
+    odml:definition "Species to which subject belongs to" ;
+    odml:dtype "string" ;
+    odml:hasValue <https://g-node.org/projects/odml-rdf#64b0c9f4-21e7-4945-9e12-342241036ca4> ;
+    odml:name "Species" .
+
+<https://g-node.org/projects/odml-rdf#64b0c9f4-21e7-4945-9e12-342241036ca4> a rdf:Bag ;
+    rdf:li "Human" .
+
+<https://g-node.org/projects/odml-rdf#79b613eb-a256-46bf-84f6-207df465b8f7> a odml:Document ;
+    odml:author "D. N. Adams" ;
+    odml:date "1979-10-12"^^xsd:date ;
+    odml:doc-version "42" ;
+    odml:hasSection <https://g-node.org/projects/odml-rdf#0f96a050-2d9b-498f-a532-fbfcc6aae55e> ;
+    odml:terminology <https://g-node.org/projects/odml-rdf#907f0698-0c25-4e43-98d8-1bb020c38b06> .
+
+<https://g-node.org/projects/odml-rdf#7b0572c1-fcfc-4010-aebf-730397ac29af> a odml:Property ;
+    odml:definition "List of crew members names" ;
+    odml:dtype "person" ;
+    odml:hasValue <https://g-node.org/projects/odml-rdf#0c60ba4e-dfcc-43f2-9a20-befcad19ba45> ;
+    odml:name "NameCrewMembers" .
+
+<https://g-node.org/projects/odml-rdf#8d68e6fe-8dda-4dfe-a1d8-edaa0f410255> a odml:Section ;
+    odml:definition "Information on the crew" ;
+    odml:hasSection <https://g-node.org/projects/odml-rdf#18bb5a7e-2097-43f2-a782-e6f612107e93> ;
+    odml:name "s12" ;
+    odml:type "crew" .
+
+<https://g-node.org/projects/odml-rdf#9b862019-643a-403f-b5a9-3b46db012961> a odml:Document ;
+    odml:author "D. N. Adams" ;
+    odml:date "1979-10-12"^^xsd:date ;
+    odml:doc-version "42" ;
+    odml:hasSection odml:bc6cc466-5769-4da1-b90e-8e5595903a4a,
+        odml:d1e70848-5318-4466-82d0-1c3e99a8ba85 ;
+    odml:terminology <https://g-node.org/projects/odml-rdf#907f0698-0c25-4e43-98d8-1bb020c38b06> .
+
+odml:bc6cc466-5769-4da1-b90e-8e5595903a4a a odml:Section ;
+    odml:definition "Information on the crew" ;
+    odml:hasSection <https://g-node.org/projects/odml-rdf#4b2d0e9d-6352-4676-97d7-fc0a795e83a9> ;
+    odml:name "s2" ;
+    odml:type "crew" .
+
+odml:d1e70848-5318-4466-82d0-1c3e99a8ba85 a odml:Section ;
+    odml:definition "Information on the crew" ;
+    odml:hasSection <https://g-node.org/projects/odml-rdf#8d68e6fe-8dda-4dfe-a1d8-edaa0f410255>,
+        odml:da908cda-19e1-4df8-b873-d1f6cd1b164d ;
+    odml:name "s1" ;
+    odml:type "crew" .
+
+odml:da908cda-19e1-4df8-b873-d1f6cd1b164d a odml:Section ;
+    odml:definition "Information on the crew" ;
+    odml:name "s11" ;
+    odml:type "crew" .
+
+<https://g-node.org/projects/odml-rdf#907f0698-0c25-4e43-98d8-1bb020c38b06> odml:Terminology <http://portal.g-node.org/odml/terminologies/v1.0/terminologies.xml> .
```

Comment:
As in the comment above, I think [this line](https://github.com/G-Node/python-odml/commit/81cc66c20d20629c7051b0d6243832c5795213a3#diff-fc2680be278b7e90cfbbdcb32f792693L79) in the `rdf_converter` also needs to be `RDF.type` as predicate.